dfusion_rust_core/src/models/mod.rs

```diff
@@ -38,6 +37,7 @@ pub trait Serializable {
 }
 
 pub trait Deserializable {
+    // TODO: encode num_tokens in the first byte(s)
```

Comment:
This is a general trait, and should probably not have a comment about the particular `num_token` issue.

Reply:
Completely agree. I think this was a merge issue... whoops.